action.yaml

```diff
@@ -54,13 +54,20 @@ outputs:
 runs:
   using: 'composite'
   steps:
+    - if: runner.os == 'Linux'
+      shell: bash
+      run: echo "LACEWORK_START_TIME=$(date --rfc-3339=seconds)" >> $GITHUB_ENV
+    - if: runner.os == 'macOS'
+      shell: bash
+      run: |
+        brew install coreutils
+        echo "LACEWORK_START_TIME=$(gdate --rfc-3339=seconds)" >> $GITHUB_ENV
```

Comment:
I think this should go before the install so that the installation is reflected in our runtime.

Reply:
Good point!

Reply:
Actually, I forgot that you need to install `coreutils` to get the `gdate` util.

Reply:
Ah okay, fine to have it after then!